arduino-ide-extension/src/browser/widgets/sketchbook/sketchbook-widget.tsx

```diff
@@ -95,15 +96,17 @@ export class SketchbookWidget extends BaseWidget {
       return;
     }
     this.sketchbookTreesContainer.activateWidget(widget);
-    const treeNode = treeWidget.model.getNode(nodeId);
+
+    const treeNode = await treeWidget.model.revealFile(new URI(nodeUri));
```

Comment:
Very nice!

If you can use the `revealFile` API, you can remove the rest of the method, can't you?

I do not think keeping
```ts
if (!SelectableTreeNode.is(treeNode)) {
      console.warn(`Tree node ${treeNode.id} is not selectable.`);
      return;
    }

    treeWidget.model.selectNode(treeNode);
```
makes sense. What do you think?

Reply:
Totally, it's not needed anymore. Thanks!